recipes/fastavro/test_recipe.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
```

Comment:
Conda does not need that. But it does not hurt if people wants to run with `./run_test.py` when debugging. (Same goes for the `build.sh` we usually keep the shebang, but conda ignores that.)